en/physics/physics-item.md

```diff
@@ -21,16 +21,62 @@ If you use __builtin__ for development, please note the following:
 
 [cannon.js](https://github.com/cocos-creator/cannon.js) is an open source physics engine, which uses js language to develop and implement more comprehensive physics functions, if your project needs more complex physical functions, then you can consider using [cannon.js](https://github.com/cocos-creator/cannon.js). The size of the __cannon.js__ module is __141KB__.
 
-## Physics Engine: ammo.js
+## Physics Engine: bullet(ammo.js)
 
 [ammo.js](https://github.com/cocos-creator/ammo.js) is the __asm.js__/__wasm__ Version of the [bullet](https://github.com/bulletphysics/bullet3) physics engine, it is compiled by [emscripten](https://github.com/emscripten-core/emscripten) tool. Bullet has perfect physical functions, and we will put more work here in the future.
 
 It should be noted that currently the __ammo.js__ module has a size of about __1.5MB__.
 
+## Physics Engine: PhysX
+
+**PhysX**（[GitHub](https://github.com/NVIDIAGameWorks/PhysX)）is an open source real-time commercial physics engine developed by Nvidia. It has perfect functional characteristics and extremely high stability, as well as excellent performance. 
+
+The **PhysX** currently supported in Cocos Creator is version 4.1 and is allowed to be used on most platforms. However, due to the large size of **PhysX** and some limitations of its own, there are some platforms Unable to get good support, these platforms are:
+
+- Various Mini Games platforms with package body restrictions (the reason is that the package body of the **PhysX** library is about 5 MB)
+- Android x86 device
+
+Some newer platforms, such as HarmonyOS and Apple M1 devices, will be supported in the future. Please pay attention to the update announcement. In addition, the ByteDance platform provides the underlying native physics, so this module can also be run in ByteDance Mini Games. For specific details, please refer to [publish ByteDance Mini Games](../editor/pulish/../publish/publish-bytedance-mini-game.md).
```

Comment:
I believe this link is wrong?

[publish ByteDance Mini Games](../editor/pulish/../publish/publish-bytedance-mini-game.md).

This looks incorrect.